packages/core/mesh/messaging/src/messenger-monitor.ts

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { trace } from '@dxos/tracing';
+
+export class MessengerMonitor {
+  public recordLongGc() {
```

Comment:
what is this metric indicative of? how would we discern a software issue vs system load?